src/AdvancedKits/Kit.php

```diff
@@ -114,7 +114,7 @@ private function loadItem(int $id = 0, int $damage = 0, int $count = 1, string $
             	if($ench instanceof CustomEnchants){
             		/** @var \PiggyCustomEnchants\Main $customEnchants */
 					$customEnchants = Server::getInstance()->getPluginManager()->getPlugin("PiggyCustomEnchants");
-					$customEnchants->addEnchantment($item, $ench,(int) $name_level);
+					$customEnchants->addEnchantment($item, $ench->getName(), (int) $name_level);
```

Comment:
Why send the name when you already have an instance of CustomEnchants?

Reply:
Because the @DaPigGuy's plugin addEnchantment is shitty AF and so on